### DISCUSSION TITLE
[Help] Tutorial for newbies

### DISCUSSION THREAD

**Discussion by mattpus** [mickael-menu/ShadowVim#49] [Help]
I got really interested in your project. I find it very enyoing that you cant change vim bindings in Xcode( I am using colmak dh layout on my keyboard). I tried to run your project but I simply dont know how to work with it. 

Would you be able to create guide from installing it to run it ?  maybe little screen recording on how you work with it ?

anyway, awesome job! 

I hope this project will get more attention in the near feature :)

**Comment by mickael-menu** [comment]
Here's a step-by-step guide, let me know if it helps:

1. Make sure you have Neovim 0.9+ installed with `nvim --version`. If not, use Homebrew to install it `brew install nvim`.
2. Drag and drop the released `ShadowVim.app` in `/Applications`.
3. Start ShadowVim, you should see that:
    <img width="344" alt="Screenshot 2023-05-04 at 21 16 13" src="https://user-images.githubusercontent.com/58686775/236307337-050d8034-399e-44b5-b68d-45cb3eb7d74c.png">
4. Click on "Open System Settings" and switch on the Accessibility permissions for ShadowVim (System Preferences > Privacy & Security > Accessibility).
    <img width="492" alt="Screenshot 2023-05-04 at 21 16 57" src="https://user-images.githubusercontent.com/58686775/236307871-9df31e4f-ec02-49df-94fc-e6eef449d65c.png">
5. Relaunch ShadowVim, or click on the Reset button in the previous pop-up. You should see the menu bar icon.
    <img width="283" alt="Screenshot 2023-05-04 at 21 19 03" src="https://user-images.githubusercontent.com/58686775/236308232-0e96be83-0a71-44b2-826e-c4bd6889119a.png">
6. Then open an Xcode project file, you should be in Normal mode by default. If not, press Escape and try the hjkl keys to move the caret around.

**Comment by mattpus** [comment]
awesome! works great. Thank you for explanation :) I would definatetly added to the main readme so people know how to try it

**Reply by mickael-menu** [reply]
Good idea, I'll update the README. Which steps did you miss when trying to install ShadowVim the first time?
